L/LLVMExtra/build_tarballs.jl

```diff
@@ -89,5 +89,5 @@ for (i,build) in enumerate(builds)
                    name, version, sources, script,
                    build.platforms, build.products, build.dependencies;
                    preferred_gcc_version=v"8", julia_compat="1.6",
-                   augment_platform_block)
+                   augment_platform_block, lazy_artifacts=true)
```

Comment:
If we always need this perhaps we should default `lazy_artifacts` to `isempty(augment_platform)`?

Reply:
If `julia_compat < v"1.7" && !isempty(augment_platform) ` but yes